elements/pl-order-blocks/pl-order-blocks.mustache

```diff
@@ -12,28 +12,26 @@ $(function() {
 
     <div class="row">
         <div class="column {{dropzone_layout}}">
-            <ul class="list-group-item list-group-item-primary header">
-                <li><strong>{{source-header}}</strong>
-                    <a role="button" class="btn btn-link btn-sm disabled" data-toggle="popover">
-                        <i class="fa fa-question-circle" aria-hidden="true" style="visibility: hidden;"></i>
-                    </a>
-                </li>
-            </ul>
-            <ul id="order-blocks-options-{{uuid}}" name="{{answer_name}}" class="pl-order-blocks-connected-sortable {{inline}}">
+            <div class="card-header list-group-item-primary">
+                {{source-header}}
+                <a style="visibility:hidden;" role="button" class="btn btn-light btn-sm" data-toggle="popover" data-html="true" title="Order Blocks" data-content="{{help_text}}" data-trigger="focus" tabindex="0">
+                    <i class="fa fa-question-circle" aria-hidden="true"></i>
+                </a>
+            </div>
+            <ul id="order-blocks-options-{{uuid}}" name="{{answer_name}}" class="card-body list-group pl-order-blocks-connected-sortable {{inline}}">
                 {{#options}}
                     <li class="{{block_formatting}} shadow" uuid="{{uuid}}" string="{{inner_html}}">{{{inner_html}}}</li>
                 {{/options}}
             </ul>
         </div>
         <div class="column {{dropzone_layout}}">
-            <ul class="list-group-item list-group-item-primary header">
-                <li><strong>{{solution-header}}</strong>
-                    <a role="button" class="btn btn-light btn-sm" data-toggle="popover" data-html="true" title="Order Blocks" data-content="{{help_text}}" data-trigger="focus" tabindex="0">
+            <div class="card-header list-group-item-primary">
```

Comment:
I know its not ideal to use bootstrap's list colors (e.g. `list-group-item-primary`) for what are supposed to be headers, but its the middle of the semester so I don't feel like we should make a huge change to the existing color scheme at the moment.

Reply:
This seems like a reasonable tradeoff.